pages/src/search/search.ts

```diff
@@ -0,0 +1,23 @@
+import {ESLBaseElement} from '../../../src/modules/esl-base-element/core';
+import {CSSClassUtils} from '../../../src/modules/esl-utils/dom/class';
+import {ready} from '../../../src/modules/esl-utils/decorators/ready';
+import {loadSearchScript} from './search-script';
+
+export class ESLDemoSearchPageWrapper extends ESLBaseElement {
+  static is = 'esl-d-search-page-wrapper';
+
+  @ready
+  protected connectedCallback(): void {
+    loadSearchScript();
+
+    (window as any).__gcse = {
+      parsetags: 'onload',
+      initializationCallback: (): void => this.afterSearchScriptLoad()
+    };
```

Comment:
Cosmetic: for better understanding and clarity of code, it is better to put this code in a separate private method, such as initSearchScript()